classes/class-wc-connect-label-reports.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+if ( ! class_exists( 'WC_Connect_Label_Reports' ) ) {
+	include_once( WC()->plugin_path() . '/includes/admin/reports/class-wc-admin-report.php' );
+
+	class WC_Connect_Label_Reports extends WC_Admin_Report {
+		/**
+		 * @var WC_Connect_Service_Settings_Store
+		 */
+		protected $settings_store;
+
+		public function __construct( WC_Connect_Service_Settings_Store $settings_store ) {
+			$this->settings_store = $settings_store;
+		}
+
+		public function get_export_button() {
+			$current_range = ! empty( $_GET['range'] ) ? sanitize_text_field( $_GET['range'] ) : '7day';
+			?>
+			<a
+				href="#"
+				download="report-<?php echo esc_attr( $current_range ); ?>-<?php echo date_i18n( 'Y-m-d', current_time( 'timestamp' ) ); ?>.csv"
```

Comment:
I know this is following the core pattern, but can we add `shipping` or `shipping-labels` to the filename?